includes/class-api.php

```diff
@@ -1104,7 +1107,7 @@ private function attach_terms( $post_id, $post_args ) {
 		if ( isset( $post_args['tax_input'] ) ) {
 
 			foreach ( $post_args['tax_input'] as $taxonomy => $terms ) {
-				wp_set_object_terms( $post_id, $terms, $taxonomy, false );
+				wp_set_object_terms( $post_id, wp_list_pluck( $terms, 'slug' ), $taxonomy, false );
```

Comment:
Previously, this method accepted `$terms`, which can be an array|int|string, but we're plucking just the slug out of that. Are there potential regressions that we haven't considered by making this change? I'm not familiar enough with what this block actually _does_ to know the impact, but want to make sure we're being cognizant of the result.

Reply:
Yeah, Why not insert the whole term array?

Reply:
The `wp_list_pluck` here is returning an array of all of the term slugs to be attached to the taxonomy. This doesn't actually insert terms that don't exist, only terms that do exist.

Reply:
so it's an update than an insert?